Misc/NEWS.d/next/Library/2022-01-24-13-00-09.bpo-46483.9XnmKp.rst

```diff
@@ -0,0 +1 @@
+Remove ``__class_getitem__`` from :class:`pathlib.PurePath`
```

Comment:
It might be good to:

1. Give a one-sentence description of why it's being removed. 
2. Delete the news entry that was added in #30822 (it'll make for a pretty confusing changelog otherwise).